scripts/py_matter_idl/matter/idl/generators/cpp/sdk/AllMetadataBridge.h.jinja

```diff
@@ -0,0 +1,35 @@
+// DO NOT EDIT MANUALLY - Generated file
+//
+// This file provides an interface to bridge the gap when doing Ember decoupling
```

Comment:
OK, this naming needs to be fixed, if it's not about bridges in the Matter sense.